index.js

```diff
@@ -343,21 +344,22 @@ app.post('/api/doorTest', async(req, res) => {
     const {locationid} = req.body;
     await redis.addDoorTestSensorData(req, res);
     await handleSensorRequest(locationid)
-});
+})
 
 app.post('/api/door', async(request, response) => {
     const coreId = request.body.coreid
-    const locationid = db.getLocationFromCoreId(coreId)
+    const location = await db.getLocationFromCoreID(coreId)
+    const locationid = location.locationid
   
     const message = JSON.parse(request.body.data)
     const signal = message.data
     var doorSignal
   
     if (signal==IM21_DOOR_STATUS.OPEN){
-        doorSignal = "open"
+        doorSignal = SESSIONSTATE_DOOR.OPEN
     }
     else if (signal==IM21_DOOR_STATUS.CLOSED){
-        doorSignal = "closed"
+        doorSignal = SESSIONSTATE_DOOR.CLOSED
     }
     else if (signal==IM21_DOOR_STATUS.LOW_BATT){
         doorSignal = "LowBatt"
```

Comment:
Should `"LowBatt"` and `"HeartBeat`" be added to `SESSIONSTATE_DOOR` enum since you plan to leave them in?